src/H5Epublic.h

```diff
@@ -322,6 +323,75 @@ H5_DLL hid_t H5Eget_current_stack(void);
  * \since 1.14.0
  */
 H5_DLL herr_t H5Eappend_stack(hid_t dst_stack_id, hid_t src_stack_id, hbool_t close_source_stack);
+/**
+ * --------------------------------------------------------------------------
+ * \ingroup H5E
+ *
+ * \brief * Check if pushing errors on an error stack is paused
+ *
+ * \estack_id{stack_id}
+ * \param[out] is_paused Flag whether stack is paused
+ * \return \herr_t
+ *
+ * \details H5Eis_paused() can be used within HDF5 VOL connectors and other
+ *          dynamically loaded components to check if the HDF5 library, or other
+ *          component has paused pushing error on the default error stack or
+ *          an application stack.
+ *
+ *          The library may pause pushing errors on the default error stack
+ *          when performing "speculative" operations, such as testing for the
+ *          existence of something that could be located at one of many
+ *          locations. \p stack_id is the error stack to query, and the value
+ *          pointed to by \p is_paused is set to TRUE/FALSE.
+ *
+ *          If an error occurs while attempting to query the status of \p stack_id,
+ *          the value pointed to by \p is_paused is unchanged.
+ *
+ * \since 1.14.4
```

Comment:
I think these will first appear in version 1.14.5, and so should be "\since 1.14.5"?

Reply:
OK, I'll update

Reply:
Done